### PR TITLE
FTC wizard & error page

### PR DESCRIPTION
**Pull request by lukefrogger**
## What's here
- The Style Review page with a resizeable iframe
- An Error page to show over the wizard when an error is encountered

### Screenshots
style-preview
Screenshot 2023-03-30 at 9 47 59 AM

### CODE REVIEW

File: packages/sitebuilder/src/components/ModalDeviceSelection.tsx [hunk 1]
```diff
@@ -2,16 +2,30 @@ import { Box, SvgIcon } from '@mui/material';
...
+	},
+	tablet: {
+		icon: TabletMac,
+		width: '768px',
+	},
+	mobile: {
+		icon: PhoneIphone,
+		width: '480px',
+	},
 };
```
Comment (chumpgrub) [inline packages/sitebuilder/src/components/ModalDeviceSelection.tsx:29]: I like the consolidation, here!

**Review by chumpgrub — APPROVED** [review]
This looks great! Nice work!

**Review by aykanburcak — COMMENTED** [review]
Great work @lukefrogger!
I'm just thinking about could it be better to name the Error screen as "Notification" or something that covers every type of warnings.
Looks like we will be also using the same UI for informing user on the successful actions such as:
https://www.figma.com/file/3DnmLW3jTWuxLVKBjQWv5c/Cloud-Platform-1.0?node-id=6532-9132&t=fsZaaTsZABpaYiVp-4

Also,  what do you think about having this screen in a higher level and make it possible to be consumed by other wizards in this (sitebuilder) package?
I'm curious about what is our plan if we need to show a similar error on `look-and-feel` wizard.
Sorry for the long comment, we can discuss this on Slack if you want.
cc @chumpgrub

**Comment by lukefrogger** [comment]
Chromatic link here: https://www.chromatic.com/pullrequest?appId=623a01cac28a2f003a843c20&number=220

**Comment by lukefrogger** [comment]
> Great work @lukefrogger! I'm just thinking about could it be better to name the Error screen as "Notification" or something that covers every type of warnings. Looks like we will be also using the same UI for informing user on the successful actions such as: https://www.figma.com/file/3DnmLW3jTWuxLVKBjQWv5c/Cloud-Platform-1.0?node-id=6532-9132&t=fsZaaTsZABpaYiVp-4
> 
> Also, what do you think about having this screen in a higher level and make it possible to be consumed by other wizards in this (sitebuilder) package? I'm curious about what is our plan if we need to show a similar error on `look-and-feel` wizard. Sorry for the long comment, we can discuss this on Slack if you want. cc @chumpgrub

**Comment by lukefrogger** [comment]
@aykanburcak I went ahead and broke up the reusable styles of the Error/Notification screen. You can access the NotificationOverlay from the components in `sitebuilder` and add anything you need to it!

**Comment by aykanburcak** [comment]
> @aykanburcak I went ahead and broke up the reusable styles of the Error/Notification screen. You can access the NotificationOverlay from the components in `sitebuilder` and add anything you need to it!

Thank you so much @lukefrogger! Looks great!

**Review by aykanburcak — APPROVED** [review]

**Pull request closed without merging**